datafusion/physical-optimizer/src/enforce_sorting/replace_with_order_preserving_variants.rs

```diff
@@ -45,7 +45,7 @@ use itertools::izip;
 pub type OrderPreservationContext = PlanContext<bool>;
 
 /// Updates order-preservation data for all children of the given node.
-pub fn update_children(opc: &mut OrderPreservationContext) {
+pub fn update_children_data(opc: &mut OrderPreservationContext) {
```

Comment:
I found the methods names hard to follow -- for example `update_children_data` actually has a `OrderPreservationContext` 🤔

Reply:
yes, I changed to `update_order_preservation_ctx_children_data`, longer but clearer